components/artwork/esd-symbol.stanza

```diff
@@ -0,0 +1,2764 @@
+#use-added-syntax(jitx)
+defpackage ocdb/esd-symbol :
+
+; Symbols: regular, large
+; Options to pass: (layer:String,show-symbol:True|False)
+; How to use:
+; create regular esd symbol on copper layer, show ESD symbol in schematic:
+;   inst <name> : ocdb/esd-symbol/regular("copper",true) 
+; create large esd symbol on silkscreen layer, show box symbol in schematic:
+;   inst <name> : ocdb/esd-symbol/large ("silkscreen",false) 
+; use place function to control position, orientation, and top/bottom layer
+  import core
+  import collections
+  import math
+  import jitx
+  import jitx/commands
+  import ocdb/defaults
+  import ocdb/land-patterns
+  import ocdb/generic-components
+  import ocdb/symbols
+
+pcb-landpattern esd-lp (lyr:LayerSpecifier|LayerIndex, side:Side|False) :
+  if side is Side :
+    layer(Courtyard(side as Side)) = Rectangle(23.368, 20.9296, loc(-2.2098, 2.6416))
+  match(lyr) :
+    (lyr:LayerSpecifier) : layer(lyr) = esd-artwork
+    (lyr:LayerIndex) : copper(lyr) = esd-artwork
+;=============================================================================
+pcb-symbol ESD-symbol (show-symbol:True|False):
+  if show-symbol :
+    layer("foreground") = esd-artwork
+  else:
+    layer("foreground") = Rectangle(5.08, 5.08, loc(0.0, 0.0))
+  layer("foreground") = Text(">VALUE", 0.762, SW, loc(-2.54, 2.84))
+  layer("foreground") = Text(">REF", 0.762, SW, loc(-2.54, 3.902))
+
+public pcb-component regular (layer:LayerSpecifier|LayerIndex,show-symbol:True|False, side:Side|False) :
+  name = "ESD Symbol"
+  description = "ESD Symbol"
+  manufacturer = "non-BOM"
+  mpn = "non-BOM"
+  assign-landpattern(esd-lp(layer, side))
+  val sym = ESD-symbol(show-symbol)
+  symbol = sym()
+  reference-prefix = "ESD"
+
+; Defaults
+public defn regular () :
+  regular(Silkscreen("artwork", Top),true, Top)
+public defn regular (layer:LayerSpecifier|LayerIndex) :
+  regular(layer,true, Top)
+public defn regular (layer:LayerSpecifier|LayerIndex,show-symbol:True|False) :

```

Comment:
I would have thought that these entrypoints should be more descriptive, like `regular-esd-symbol()`.
Now that I see their usage in `ble-mote`, it reads better like this.
These should be always accessed with qualified packages `ocdb/esd-symbol/regular()` because of common name collisions with other packages.

i.e. The following requires prefixing:
```
import ocdb/esd-symbol
; --> Use prefix ---- with: prefix(large) => esd-symbol-
import ocdb/jitx-logo
; --> Use prefix ---- with: prefix(large) => jitx-logo-

esd-symbol-large()
```

Is it the expected workflow for a designer to write `ocdb/esd-symbol/regular()` instead of importing `ocdb/esd-symbol`?

Reply:
Exactly, typically, components in ocdb are named "component". here I named them small / large to describe size. It's not expected to add a dedicated import statement for any ocdb component, but if someone did on these or most components in ocdb, they would likely need a prefix to avoid collisions with other components. General workflow is for designers to use fully qualified names:
`  inst pcb-name: ocdb/board-text/text("PCB, BLE MOTE", 1.5, clearance(current-rules(), MinSilkscreenWidth) )
  inst logo : ocdb/jitx-logo/small(Silkscreen("artwork", Top),true)
  inst assy-label : ocdb/mechanical/gen-label()`